src/Illuminate/View/Concerns/ManagesLoops.php

```diff
@@ -69,11 +69,13 @@ public function popLoop()
     /**
      * Get an instance of the last loop in the stack.
      *
-     * @return array
+     * @return \stdClass
```

Comment:
`\stdClass|null` (PHP turns nothing into `null` when a function is called. &lt;rant&gt;Their so called `void` type is so misleading, because it is still returning `null` in effect, so having it as a return type is silly, since all it's really doing is enforcing any return statements must have no operands&lt;/rant&gt;)